install-python-requirements

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+usage() {
+  echo "Create a Python virtual environment with dependencies from requirements.txt"
+  echo "Usage: $(basename "$0") <requirements.txt> <venv-dir>"
+}
+
+if [[ $# -ne 2 ]]; then
+  usage
+  exit 1
+fi
+
+requirements="$1"
+venv_dir="$2"
+
+# Create virtual environment
+mkdir -p "$venv_dir"
+# Note: The --copies flag is important here because it tells venv to copy
+# rather than symlink in library files, which is important for the virtual
+# environment to function properly in a sandboxed Wake execution.
+python3.7 -m venv --copies "$venv_dir"
```

Comment:
I'm wondering if it might be easier long term if we only specify `python3` rather than 3.7, then we can let `setPlanResources` handle which python version runs from one spot

Reply:
I'd be happy to add a commit to do this, but I'm not sure where (or if) to put a `setPlanResources` here.

Reply:
The reason we explicitly listed 3.7 is because we've got a bit of technical debt that requires us to have both Python 3.5 (the Ubuntu 16.04-provided one) and Python 3.7 installed simultaneously. In several of our environments, I think we left `python3` to map to the distribution-provided Python and forced ourselves to invoke `python3.7` to refer to our custom Python 3.7.

I agree longer term that we should not depend on having the minor version component present, but it might be better to just leave this here and overhaul the whole environment system in a different way. Longer term, we need to more precisely describe the "contract" of the different environments, and, for example, explicitly specify whether `python3` when invoked within a `python/python/3.7.1` environment refers to the Python 3.7 or the distro-provided Python.

If you can make `python3` (sans `.7` suffix) work in all of the environments we currently support, then I'm all for leaving out the `.7`, but if that isn't easy to do, then I don't think we need to block on that.